pkg/config/config.go

```diff
@@ -37,6 +37,7 @@ type BoxTextsOption struct {
 	BoxSpacing *int      `json:"boxSpacing,omitempty"`
 	BoxAlign   box.Align `json:"boxAlign,omitempty"`
 	Enabled    *bool     `json:"enabled,omitempty"`
+	Limit      *int      `json:"limit,omitempty"`
```

Comment:
I prefer using values instead of pointers as possible. `Limit` can use the initial zero value as a disabled value, so we can use `int` instead of the pointer.